cfpb/consumerfinance.gov
# Fix some outdated Apache redirects to filter pages

Our Apache redirects contain many target URLs that include outdated query strings for blog pages, using `?filter0_` or `?filter1_` prefixes. These prefixes were removed from code in #3603, but these redirects were inadvertently left behind.

This commit corrects some but not all of these redirects, partially addressing internal el-camino#316. The rest of the `?filter` redirect fixes will come in a subsequent PR.

## How to test this PR

To test, run a server with Apache in the preferred method of your choosing, and verify that URLs redirect as expected.

## Checklist
